extensions/cache/deployment/src/main/java/io/quarkus/cache/deployment/CacheConfig.java

```diff
@@ -79,6 +79,12 @@ public static class CaffeineNamespaceConfig {
              * value to {@code true} will enable the accumulation of cache stats inside Caffeine.
              */
             boolean metricsEnabled;
+
+            /**
+             * Whether or not null values are put to cache
+             */
+            @ConfigItem
+            boolean cacheNullValues = true;
```

Comment:
This will make the default value appear in the doc which is automatically generated.
```suggestion
            @ConfigItem(defaultValue = "true")
            boolean cacheNullValues;
```